DOCS/man/vf.rst

```diff
@@ -247,12 +247,13 @@ Available mpv-only filters are:
         :bt.601-525:   ITU-R BT.601 (SD) 525-line systems (NTSC, SMPTE-C)
         :bt.601-625:   ITU-R BT.601 (SD) 625-line systems (PAL, SECAM)
         :bt.709:       ITU-R BT.709 (HD) (same primaries as sRGB)
-        :bt.2020:      ITU-R BT.2020 (UHD)
+        :bt.2020:      ITU-R BT.2020 (UHD), default for HDR transfers
```

Comment:
This is not true, mpv does not default the primaries based on the transfer

Reply:
Yes, that is jot mpv comment though.